Sources/Views/Products/Detail/EcoscoreImageView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  EcoscoreImageView.swift
+//  OpenFoodFacts
+//
+//  Created by arnaud on 21/11/2020.
+//
+
+import UIKit
+
+class EcoscoreImageView: UIImageView {
+
+    public enum Ecoscore: String {
+        case ecoscoreA = "a"
+        case ecoscoreB = "b"
+        case ecoscoreC = "c"
+        case ecoscoreD = "d"
+        case ecoscoreE = "e"
+        case unknown
+    }
+
+    /// return the currently displayed nova group
+    public var ecoScore: Ecoscore = .ecoscoreA {
```

Comment:
maybe default to `unknown` instead of `ecoscoreA` ?
Also the comment might need a rename from nova to ecoscore

Reply:
Yes, good idea.

Reply:
done